### PR TITLE
UX-663 Takeover design tweaks

### LINKED ISSUE #295: Takeover Design Tweaks
### Design tweaks

**Header**
1. Remove `border-bottom: 1px solid ${tokens.border.color};` on the header Wrapper
2. Heading should default to `level={3}` and `isLight`. Also, it seems like the styles on lines 25-33 in Header.styles.js are overriding the heading font size and resetting the font-size to `16px`:
```
  .heading--level-1,
  .heading--level-2,
  .heading--level-3,
  .heading--level-4,
  .heading--level-5 {
    ${fontSize()};
    font-weight: 400;
    margin: 0;
  }
```
3. The close button needs to be completely reworked. I have included some screenshots below:
    1.  Padding on the header Wrapper should be `padding: 0 0 0 ${${spacer(2)}` so that the close button container `div` can be positioned in the far right of the header
    2. The close button container `div` should be `48px x 48px` = `width: ${spacer(6)}` and `height: ${spacer(6)}`
    3. The close button container should have `border-left: 1px solid ${tokens.border.color}`
    4. Close button should be `size="medium"`
close button specs
4. The Header should also be on a higher `z-index` than the Content so that when the page is scrolled the Content will scroll "under" the Header.

**Content**
1. Remove `border: 1px solid ${tokens.border.color}` on the Content Wrapper
2. Change the border-radius to `border-radius: ${tokens.card.borderRadius}`
3. Change the box-shadow to `box-shadow: 0 0 0 1px rgba(63, 61, 60, 0.05), 0 1px 3px 0 rgba(63, 61, 60, 0.15);`   Let's go ahead and create a Paprika token for this shadow value and call it `${tokens.card.shadow}`

So these are the styles that I wrote for the Content wrapper:
```
  background-color: ${tokens.color.white};
  border-radius: ${tokens.card.borderRadius};
  box-shadow: 0 0 0 1px rgba(63, 61, 60, 0.05), 0 1px 3px 0 rgba(63, 61, 60, 0.15);
  margin: ${spacer(2)};
  padding: ${spacer(2)};
```

**Here is a screenshot of what I have locally for ALL the above changes:**
Screen Shot 2019-11-21 at 11 16 48 AM

That should be all for now. Thanks!

### PR DESCRIPTION
**Pull request by DanilAgafonov**
### Purpose 🚀
Some design tweaks for Takeover component requested by @johnpeele 

Resolves #295 

### Notes ✏️
_details of code change / secondary purposes of this PR_

### Updates 📦
- [x] MAJOR (breaking) change to `@paprika/takeover`
- [x] MINOR (backward compatible) change to `@paprika/tokens`
- [ ] PATCH (bug fix) change to _these packages_

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/UX-663-takeover-design-tweaks

### Screenshots 📸
**Before:**
paprika highbond com_iframe html_id=takeover--basic (1)

**After:**
localhost_9009_iframe html_id=takeover--basic

### References 🔗
https://aclgrc.atlassian.net/browse/UX-663
https://github.com/acl-services/paprika/issues/295

### CODE REVIEW

**Comment by nahumzs** [comment]
this seems good to me, let's wait and see if some else has a comment.

File: packages/Takeover/src/components/Content.js [hunk 1]
```diff
@@ -4,9 +4,8 @@ import tokens from "@paprika/tokens/lib/tokens";
 
 const Content = styled.div`
   background-color: ${tokens.color.white};
-  border: 1px solid ${tokens.border.color};
   border-radius: ${tokens.border.radius};
```
Comment (johnpeele) [inline packages/Takeover/src/components/Content.js]: @DanilAgafonov This border radius should be `border-radius: ${tokens.card.borderRadius}`

Comment (DanilAgafonov) [reply packages/Takeover/src/components/Content.js]: Hmmm, looks like we are trying to create card/paper component.

@johnpeele what do you think, if I create `@paprika/paper` component? Something like https://material-ui.com/components/paper/

**Comment by oscarkwan** [comment]
Merged for you! @DanilAgafonov

**Pull request merged**